apps/web/src/hooks/useCakeDistributed.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query'
+import { useRevenueSharingPoolForCakeContract } from './useContract'
+
+export const useCakeDistributed = (): bigint => {
+  const revenueSharingPoolContract = useRevenueSharingPoolForCakeContract()
+
+  const { data } = useQuery(
+    ['cakeDistributed', revenueSharingPoolContract.address],
```

Comment:
double checking if it has any chains difference

Reply:
chain id should be part of the key